scout/load/panel.py

```diff
@@ -160,7 +162,7 @@ def load_panelapp_panel(adapter, panel_id=None, institute="cust000", confidence=
 
     if not panel_id:
         LOG.info("Fetching all panel app panels")
-        panel_ids = _panelapp_panel_ids()
+        panel_ids: List[str] = _panelapp_panel_ids()
```

Comment:
To me the type hint is redundant, since you give it on the function definition, but if you think it helps reading, it's fine!

Reply:
When working on this code I had to go and check that function to understand the return type, so I think it would be good to keep it. It helps I think

Reply:
Hihi, ok! PyCharm is kind enough to show those to me anyway. 😁